internal/operator/subscription.go

```diff
@@ -25,13 +25,13 @@ func Subscriptions(catalogSource string, catalogSourceNamespace string) ([]Subsc
 
 	c, err := NewPackageServerClient()
 	if err != nil {
-		logger.Errorf("Error while creating new PackageServerClient: %s", err)
+		logger.Errorw("Error while creating new PackageServerClient", err)
 		return nil, err
 	}
 
 	packageManifests, err := c.OperatorsV1().PackageManifests("").List(context.Background(), metav1.ListOptions{})
 	if err != nil {
-		logger.Errorf("Error while listing new PackageManifest Objects : %s", err)
+		logger.Errorw("Error while listing new PackageManifest Objects", err)
```

Comment:
Errorf instead of Errorw? If Errorw need to fix format; logger.Errorw("Error while creating new PackageServerClient", "error", err)